### ISSUE TITLE
Losses rise

### ISSUE THREAD

**Issue by wly-ysu** [zhyever/Monocular-Depth-Estimation-Toolbox#33]
Hello, due to equipment problems, I need to modify sample_per_gpu, for example, 8 in the ./configs/adabins/adabins_efnetb5ap_kitti_24e.py file is modified to 1, but during training, the loss will rise around 20k, how should I set the parameters to improve the accuracy, looking forward to your reply.

**Comment by zhyever** [comment]
Turn down the learning rate?

**Comment by wly-ysu** [comment]
Thank you for your guidance, I modified the learning rate, the loss did not increase during training, but the indicator can not reach the indicator in your log file, I think my learning rate should be increased, because this time the learning rate is probably small ten times.
